README.md

```diff
@@ -251,7 +251,7 @@ make formatter
 
 #### Type Checking
 
-If you want to check types on the codebase, install `pytype` using `poetry install`.
+If you want to check types on the codebase, install `mypy` using `poetry install`.
```

Comment:
should we keep the pytype check until we made enough progress on mypy? I am bit worried about the period of time where we disabled a lot of mypy checks and are also not running pytype

Reply:
yes this is a concern I have too 👍 I will see if there are issues if we keep both of them at the same time (for instance, there might be problems with `# pytype: disable` comments)

Reply:
17f125d ✅